src/App.tsx

```diff
@@ -1,19 +1,147 @@
 import React from 'react';
-import './App.css';
+import './App.scss';
+import { TodoList, Todo } from './components/TodoList';
 
 import users from './api/users';
+import todos from './api/todos';
 
-const App: React.FC = () => {
-  return (
-    <div className="App">
-      <h1>Add todo form</h1>
-
-      <p>
-        <span>Users: </span>
-        {users.length}
-      </p>
-    </div>
-  );
-};
+export interface State {
+  todos: Todo[],
+  user: string,
+  title: string,
+  hasTitleError: boolean,
+  hasUserError: boolean,
+}
+
+export class App extends React.Component {
+  state: State = {
+    todos: [...todos],
+    user: '',
+    title: '',
+    hasTitleError: false,
+    hasUserError: false,
+  };
+
+  handleUserChange = (event: React.ChangeEvent<HTMLInputElement | HTMLSelectElement>) => {
```

Comment:
You don't need union type here any more.
```suggestion
  handleUserChange = (event: React.ChangeEvent<HTMLSelectElement>) => {
```